packages/@react-native-windows/codegen/package.json

```diff
@@ -32,7 +32,6 @@
     "@rnw-scripts/just-task": "2.2.3",
     "@rnw-scripts/ts-config": "2.0.2",
     "@types/chalk": "^2.2.0",
-    "@types/globby": "^9.1.0",
```

Comment:
Just double checking that this is intentional and that this package is not needed anymore.

Reply:
It looks like newer versions of globby provide their own typings.

Reply:
yup, checked for .d.ts file now being inside globby